sandbox/simulator/randomtext

```diff
@@ -288,3 +302,44 @@ function MarkovRandomPhrase($minwords = 3, $maxwords = 25)
 	return implode(" ",$words);
 }
 
+/// Generate 'Random' Youtube Video
+
+function YoutubeRandomVideo()
+{
+	global $useyoutubevideos;
+
+	if ($useyoutubevideos)
+	{
+
+		$url = "https://www.youtube.com/watch?v=";
+
+		// Using a fixed pool of YT videos (LudumDare keynotes)
+		$videoids = [
+			1 => "Hrkv31Lep9s",
```

Comment:
Suggest omitting the keys here as a good practice. Also as it is coded below, $index could be 0, and that would cause an error due to missing 0 key.